lib/retriable.rb

```diff
@@ -0,0 +1,13 @@
+module Retriable
+  def retries(*rescue_errors)
+    retries ||= 0
+    yield if block_given?
+  rescue *rescue_errors, StandardError
```

Comment:
This changes several locations which previously only rescued Faraday::Error and UnparseableResponseError to _also_ rescue from `StandardError` which is much broader in scope and generally best avoided

I think this needs to either explicitly require a list of exceptions, and check the list isn't empty or just default the list to `Faraday::Error` when its empty - so either

```
rescue_errors = [Faraday::Error] if rescue_errors.empty?
retries ||= 0
yield if block_given?
rescue *rescue_errors
```

Reply:
AFAIK, its common practice to rescue StandardError. I think you might be mixing it with the principle of not rescuing Exception?

Reply:
Since the production, code uses `StandardError`,  and this is just a refactor,
and this retries is general purpose,  I'm gonna use the same `StandardError`.

The only change I am doing is
`rescue_errors << StandardError if rescue_errors.empty?`

(potentially the logic is the same, it's just more explicit)